website/docs/table-group/data-table/data-table-code.md

```diff
@@ -375,45 +402,40 @@ Add a skeleton to the table by directly substituting it in the `data` or replaci
 
 :::
 
-## Columns merging
+## Specific cases
```

Comment:
if we move all the accordion examples out of here, we can rename this to "Custom elements in table" or something, then it will be much easier to find